src/screencast_keys/preferences.py

```diff
@@ -228,9 +231,12 @@ class SK_Preferences(bpy.types.AddonPreferences):
         max=100,
     )
 
+    # Increase default font size on macOS (Retina) for better visibility
     font_size: bpy.props.IntProperty(
         name="Font Size",
-        default=int(bpy.context.preferences.ui_styles[0].widget.points),
+        default=int(
+            bpy.context.preferences.ui_styles[0].widget.points
+            * (5 if is_macos else 1)),
```

Comment:
I think changing `font_size` is not good idea because this configuration should be constraint in user side.
Instead, we can change the scale before feeding the font_size to API.
This is same for others.